From 6592d7f0ba3da5635c81ac1f5e895a13ad1ccda1 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Tue, 24 Apr 2018 11:47:02 +0800
Subject: [PATCH] arm64: rockchip_linux_defconfig: enable scsi scan async

Enable CONFIG_SCSI_SCAN_ASYNC to probe device on
different buses in parallel, it can speed up the
USB 3.0 UAS disk enumeration on rockchip platform.

Change-Id: I30afc13928c8ea6eac16e3a96ef1f79b6bda56ae
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index 194e45ac6414..71c17313c1a7 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -118,6 +118,7 @@ CONFIG_SRAM=y
 CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_DEV_SR=y
+CONFIG_SCSI_SCAN_ASYNC=y
 CONFIG_SCSI_SPI_ATTRS=y
 CONFIG_MD=y
 CONFIG_NETDEVICES=y
-- 
2.35.3

